cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/BrokerMetricSample.java

```diff
@@ -166,10 +308,25 @@ public double allTopicsFetchRequestRate() {
    * 8 bytes - all topics produce request rate
    * 8 bytes - all topics fetch request rate
    * 8 bytes - sample time
+   * 8 bytes - broker request queue size
```

Comment:
Probably does not matter much in our case as we won't have many brokers. But the queue size and time can be int. Unlike request rate/bytes in rate, they are unlikely to exceed the integer range.